contribute-to-docs/docs-release-process.md

```diff
@@ -55,7 +55,7 @@ to include the new version, and remove the oldest. Order matters, most recent fi
 
 ## Update the dot-release Job
```

Comment:
I just remembered, you probably need to add this as another step on line 12 above.